app_mobilenet_alpha_1/Makefile

```diff
@@ -0,0 +1,31 @@
+TARGET=XCORE-AI-EXPLORER
+APP_NAME=app
```

Comment:
Please rename the folder name to `app_mobilenetv2_alpha_1` from `app_mobilenet_alpha_1`. Could also change the `APP_NAME` to that.